autofill/autofill-impl/src/main/AndroidManifest.xml

```diff
@@ -21,7 +21,7 @@
         <activity
             android:name="com.duckduckgo.autofill.ui.credential.management.AutofillManagementActivity"
             android:configChanges="orientation|screenSize"
-            android:windowSoftInputMode="stateVisible|adjustResize"
```

Comment:
I'm unsure why we had this here; it looks to have been added for editing credentials but I see that working fine without it. 

I'm removing it because it causes the keyboard to automatically show when entering the credential list on some devices (where it doesn't make sense to present a keyboard)

Reply:
You are right. This seems to be not necessary.

Reply:
Without it, the previously reported issue seems to be fixed still.